.github/workflows/r-cmd-check.yml

```diff
@@ -34,6 +30,7 @@ jobs:
       fail-fast: false
       matrix:
         config:
+         # - {os: windows-latest, r: '4.2', repos: 'https://cran.microsoft.com/snapshot/2022-11-01/'}
           - {os: ubuntu-20.04, r: '4.2', repos: 'https://cran.microsoft.com/snapshot/2022-11-01/'}
```

Comment:
Why don't you add the Windows check here instead of creating a new file?

Reply:
ALso can you upgrade the snapshot to `2022-12-14`?

Reply:
Sure happy to upgrade.  The two weren't playing nice together (linux and windows) so I just split them out.  Definitely, a better way to do it , i.e. put them together, just didn't figure it out.